app/experimenter/nimbus-ui/src/components/TableOverview/index.stories.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { RouterSlugProvider } from "../../lib/test-utils";
+import { withLinks } from "@storybook/addon-links";
+import { mockExperimentQuery } from "../../lib/mocks";
+import TableOverview from ".";
+// import { mockAnalysis } from "../../lib/visualization/mocks";
```

Comment:
keep/remove?

Reply:
Keeping, we'll need it when we build the TableOverview out. https://github.com/mozilla/experimenter/pull/3973#discussion_r524815256